spread.yaml

```diff
@@ -164,7 +164,13 @@ backends:
             - centos-8-64:
                   workers: 6
                   storage: preserve-size
-                  image: centos-stream-8
+                  image: centos-8-64
```

Comment:
Those are our images based on centos-stream-8 from GCE?

Reply:
centos-stream-8 is the upstream image, that one is broken and it has not the test dependencies installed
centos-8-64 is our image which has the test dependencies installed